_docs-sources/guides/stay-up-to-date/patcher/index.md

```diff
@@ -0,0 +1,74 @@
+# Patcher
+
+## Installation
+
+To install Patcher, download the appropriate package for your system from the [releases](https://github.com/gruntwork-io/patcher-cli/releases) page.
+
+### For MacOS and Linux users
+
+Patcher runs in single binary called `patcher`. Make sure it's available within your `PATH`.
+
+1. After downloading Patcher, unzip the package (only MacOS releases are zipped). Patcher runs in a single binary.
```

Comment:
> Patcher runs in a single binary
Do we need to add that context? I think it may be more clear to say something like:

Suggestion:
"After downloading Patcher, move it to your desired destination directory. If you're using MacOS, you will have to unzip the package. For other operating systems, the artifact you download is the executable itself."

Reply:
Yes, that sounds good! Thanks